org-formation/600-access/_tasks.yaml

```diff
@@ -9,13 +9,29 @@ AgoraCIServiceAccount:
   Parameters:
     ManagedPolicyArns:
       - arn:aws:iam::aws:policy/AdministratorAccess
+      - - arn:aws:iam::aws:policy/PowerUserAccess
   DefaultOrganizationBinding:
     IncludeMasterAccount: false
     Account:
       - !Ref AgoraDevAccount
       - !Ref AgoraProdAccount
     Region: us-east-1
 
+    # Setup Agora collaborator access
+    AgoraCollaboratorAccess:
+      Type: update-stacks
+      Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.10/templates/IAM/cross-account-access.yaml
+      StackName: agora-collaborator-access
+      Parameters:
+        PrincipalArns:
+          - arn:aws:iam::155057775224:user/MattC    # Matthias Courville from GenUI
+        ManagedPolicyArns:
+          - arn:aws:iam::aws:policy/PowerUserAccess
```

Comment:
PowerUserAccess is nearly the same as Admin which is probably too permissive for a collaborator.  We prefer to scope collaborator permissions to least privilege.  The question is what do you intend for Matthais to do in the AWS account?  Just view Beanstalk configuration and logs, access EC2 instances, access s3 buckets, deploy the agora app to EBS, make changes to the beanstalk environment?  or the entire kitchen sink?